onos-umbrella/Chart.yaml

```diff
@@ -28,7 +28,7 @@ dependencies:
   - name: onos-cli
     condition: import.onos-cli.enabled
     repository: file://../onos-cli
-    version: 1.3.13
+    version: 1.3.14
```

Comment:
make sure you bump onos-umbrella chart version as well.

Reply:
Yep. :)